stacks-signer/src/main.rs

```diff
@@ -124,8 +124,11 @@ fn process_dkg_result(dkg_res: &[OperationResult]) {
                 &schnorr_proof.r, &schnorr_proof.s,
             );
         }
-        OperationResult::DkgError(..) | OperationResult::SignError(..) => {
-            todo!()
+        OperationResult::DkgError(dkg_error) => {
+            panic!("Received DkgError {}", dkg_error);
```

Comment:
could this be a DOS vector? is it possible to craft a scenario to trigger this panic?

Reply:
All unexpected responses `panic` the current `stacks-signer` binary.  This will need to be addressed at some point, but is orthogonal to this PR.

Reply:
got, thanks!